api/src/main/resources/soknadsmottaker-api.yml

```diff
@@ -34,6 +36,48 @@ paths:
       tags:
         - soknad
 
+
+  /health/ping:
+    get:
+      summary: Pings the application to see if it responds
+      description: Pings the application to see if it responds
+      operationId: ping
+
+      responses:
+        '200':
+          description: Successful operation; application is responding
+      tags:
+        - health
+
+  /health/isAlive:
+    get:
+      summary: Checks if the application and its dependencies up
+      description: Checks if the application and its dependencies are up and running.
+      operationId: isAlive
+
+      responses:
+        '200':
+          description: Successful operation; application is alive
+        '500':
+          description: The application or one of its dependencies are not up and running.
+      tags:
+        - health
+
+  /health/isReady:
+    get:
+      summary: Checks if the application is ready to accept traffic
+      description: Checks if the application is ready to accept traffic.
+      operationId: isReady
+
+      responses:
+        '200':
+          description: Successful operation; application is ready
+        '503':
+          description: The application or one of its dependencies are not ready
+      tags:
+        - health
+
```

Comment:
Alle API-endepunktene er i følge denne definisjonen beskyttet av basic auth:
```
  securitySchemes:
    basicAuth:
      type: http
      scheme: basic

```
Det er ikke riktig for health/.... endepunktene. 
Samme problemet eksisterer for soknadsfillager

Reply:
Bra funn! Jeg har pushet commit som jeg trur skal fikse det

https://swagger.io/docs/specification/authentication/